index.html

```diff
@@ -0,0 +1,164 @@
+        <div class="upper-layout">
+            <ul class="upper-group">
+                <li class="upper-items"><i class="fab fa-facebook-f"></i></li>
+                <li class="upper-items"><i class="fab fa-twitter"></i></li>
+                <li class="upper-items">English</li>
+                <li class="upper-items">My Page</li>
+                <li class="upper-items">Logout</li>
+            </ul>
+        </div>
+
+        <div class="nav-bar">
+            <a href="index.html"><img class="sabre-logo" src="images/sabre-awards-logo.jpg" alt="sabre-logo"></a>
+            <ul class="nav-group">
+                <li class="nav-items"><a id="nav-about" class="nav-links" href="about.html">About</a></li>
+                <li class="nav-items"><a class="nav-links" href="#">Program</a></li>
+                <li class="nav-items"><a class="nav-links" href="#">Join</a></li>
+                <li class="nav-items"><a class="nav-links" href="#">Sponsor</a></li>
+                <li class="nav-items"><a class="nav-links" href="#">News</a></li>
+                <li class="nav-items"><a id="nav-border" class="nav-links" href="#">SABREs Campaign</a></li>
+            </ul>
+        </div>
+        
+
+        <!-- Navigation for Mobile menu -->
+        <div class="nav-mobile">
+            <ul class="nav-menu">
+                <li><a class="nav-link" id="menu-x" href="#">X</a></li>
+                <li><a class="nav-link" href="about.html">About</a></li>
+                <li><a class="nav-link" href="#about">Program</a></li>
+                <li><a class="nav-link" href="#contact">Join</a></li>
+                <li><a class="nav-link" href="#contact">Sponsor</a></li>
+                <li><a class="nav-link" href="#contact">News</a></li>
+                <li><a class="nav-link" href="#contact">SABREs Campaign</a></li>
+            </ul>
+        </div>
+
+    </header>
+
+    <main>
+
+        <!-- Details section -->
+        <section id="details-section">
+
+            <div class="hamburger">
+                <span class="bars"></span>
+                <span class="bars"></span>
+                <span class="bars"></span>
+            </div>
+
+            <div class="details-info">
+                <h1 class="details-text-one">"Hello! Sharing Heroes"</h1>
+                <h2 class="details-heading">GLOBAL SABRE INTERNATIONAL AWARDS WINNERS 2021</h2>
+                <p class="details-paragraph">The SABRE Awards — which include competitions for the Americas, EMEA, Latin America and the Asia-Pacific region — attract more than 5,500 entries from more than 60 countries.</p>
+                <h3 class="details-date">2022.09.10 (SAT) ~ 13(MON)</h3>
+                <h3 class="details-text-two">@ Hampton Inn Manhattan-Times Square North</h3>
+            </div>
+        </section>
+
+        <!-- Main Program section -->
+        <section class="program-section">
+            <div class="main-program-header">
```

Comment:
- [ ] In the desktop screen, please fix the alignment of the elements because they're overflowing on each other.
![Screenshot (1)](https://user-images.githubusercontent.com/47625407/176824573-9a4e0d55-55ff-46af-89ac-58a18cdbd5b4.png)